### PR TITLE
feat(console): table loading component

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add table loading component.

## Linear Issue Reference

LOG-1968

## Testing

Mar-23-2022 14-42-42

### CODE REVIEW

File: packages/console/src/pages/Users/index.tsx [hunk 1]
```diff
@@ -65,9 +66,17 @@ const Users = () => {
...
+              <td className={styles.userName}>
+                <ItemPreviewLoading />
+              </td>
+              <td>
+                <div />
+              </td>
+              <td>
+                <div />
+              </td>
+            </TableLoading>
```
Comment (xiaoyijun) [inline packages/console/src/pages/Users/index.tsx:79]: the usage of this `TableLoading` seems verbose, any good idea? @logto-io/eng

**Review by xiaoyijun — APPROVED** [review]
Overall LGTM.

File: packages/console/src/pages/Users/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/Users/index.tsx:79]: The main reason is that every table may have different number of columns, and each column's loading block may be different as well.

Comment (wangsijie) [reply packages/console/src/pages/Users/index.tsx:79]: Also, column width is customized by classNames.

Comment (demonzoo) [reply packages/console/src/pages/Users/index.tsx:79]: I think we should not control the column width using css class names on each column. Should use `<col />` instead, IMO. Then we can avoid defining the <td>s in the loading skeleton.

**Pull request merged**